pkg/client/restconfig/restconfig.go

```diff
@@ -49,10 +49,24 @@ var configSources = []configSource{
 	},
 }
 
+// NewRestConfigWithThrottling will attempt to create a new rest config from all
+// configured options and return the first successfully created configuration.
+// The flag restConfigSource, if specified, will change the behavior to attempt
+// to create from only the configured source.
+// It forces to enable the client-side throttling to fix the commit stuck in the
+// inProgress state issue.
+func NewRestConfigWithThrottling(timeout time.Duration) (*rest.Config, error) {
+	return newRestConfig(timeout, true)
+}
+
 // NewRestConfig will attempt to create a new rest config from all configured options and return
 // the first successfully created configuration.  The flag restConfigSource, if specified, will
```

Comment:
This function does not have a flag `restConfigSource`. The comment needs to be udpated.

Reply:
DONE